polaris_pdns/core/polaris.py

```diff
@@ -222,9 +223,9 @@ def _any_response(self, params, qname):
         # determine how many records to return
         # which is the minimum of the dist table's num_unique_addrs and 
         # the pool's max_addrs_returned
-        if dist_table['num_unique_addrs'] <= pool['max_addrs_returned']: 
+        if dist_table['num_unique_addrs'] <= pool['max_addrs_returned']:
             num_records_return = dist_table['num_unique_addrs']
-        else:    
```

Comment:
Why are you changing white spaces ? Event if it is a valid change.

It unfocuses the code review.

Reply:
Please, can you rebase your serie with only the relevant changes for this ends client subnet feature ?